doc builds failing on 0.11.0 (KeyError in tracer)

Attempting to release 0.11.0, doc builds were failing with:

```
make html
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v4.0.0
/opt/miniconda3/envs/xonshdocs/lib/python3.9/site-packages/cloud_sptheme/__init__.py:78: DeprecationWarning: get_theme_dir() is deprecated, and will be removed in version 2.0
  warn("get_theme_dir() is deprecated, and will be removed in version 2.0",
WARNING: The config value `autosummary_generate' has type `list'; expected `bool'.
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 134 source files that are out of date
updating environment: [new config] 134 added, 0 changed, 0 removed
reading sources... [  1%] aliases                                                                                                                                                                                    
Exception occurred:
  File "/home/gil/github.com/xonsh/xonsh/xonsh/tracer.py", line 225, in __call__
    spec = kwargs["spec"]
KeyError: 'spec'
The full traceback has been saved in /tmp/sphinx-err-f691khg5.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make: *** [Makefile:39: html] Error 2
```

Until I hack-patched it thus:
```
diff --git a/xonsh/tracer.py b/xonsh/tracer.py
index 3457f241..fb53f0fe 100644
--- a/xonsh/tracer.py
+++ b/xonsh/tracer.py
@@ -222,11 +222,14 @@ class TracerAlias(ArgParserAlias):
         return parser
 
     def __call__(self, *args, **kwargs):
-        spec = kwargs["spec"]
-        usecolor = (
-            spec.captured not in xpp.STDOUT_CAPTURE_KINDS
-        ) and sys.stdout.isatty()
-        tracer.color_output(usecolor)
+        try:
+            spec = kwargs["spec"]
+            usecolor = (
+                spec.captured not in xpp.STDOUT_CAPTURE_KINDS
+            ) and sys.stdout.isatty()
+            tracer.color_output(usecolor)
+        except KeyError:
+            pass
         return super().__call__(*args, **kwargs)
 ```

--- a/xonsh/tracer.py
+++ b/xonsh/tracer.py
@@ -222,9 +222,9 @@ class TracerAlias(ArgParserAlias):
         return parser
 
     def __call__(self, *args, **kwargs):
-        spec = kwargs["spec"]
+        spec = kwargs.get("spec")
         usecolor = (
-            spec.captured not in xpp.STDOUT_CAPTURE_KINDS
+            spec and (spec.captured not in xpp.STDOUT_CAPTURE_KINDS)
         ) and sys.stdout.isatty()
         tracer.color_output(usecolor)
         return super().__call__(*args, **kwargs)

